packages/tasks/src/tasks/image-to-image/about.md

```diff
@@ -53,13 +52,22 @@ await inference.imageToImage({
 });
 ```
 
-## ControlNet
+## Use Cases for Text-to-Image Tasks
```

Comment:
this will confuse user as there's a separate task called `text-to-image`maybe rename it to be something like Image Generation?

Reply:
I feel like it's best to have ## Use Cases and then a subsection called ### Use Cases for Image Generation

Reply:
yeah I see what you mean, the idea here was to follow up on the connection to text-to-image as was mentioned in the task summary. 
I think tho that ### Use Cases for Image Generation is a bit confusing too since it's not unique to text-to-image with added visual guidance and can also be understood as the uses cases above
maybe `Uses Cases for Text Guided Image Generation` ?

Reply:
yes it works, you might want to say Image and Text Guided Image Generation not to confuse lol